core/src/test/java/org/elasticsearch/search/simple/SimpleSearchIT.java

```diff
@@ -336,6 +337,19 @@ public void testTooLargeFromAndSizeBackwardsCompatibilityRecommendation() throws
                 .setFrom(DefaultSearchContext.Defaults.MAX_RESULT_WINDOW * 10).get(), 1);
     }
 
+    public void testQueryNumericFieldWithRegex() throws Exception {
+        createIndex("idx");
+        indexRandom(true, client().prepareIndex("idx", "type")
+                .setSource(XContentFactory.jsonBuilder().startObject().field("num", 34).endObject()));
```

Comment:
In this case you can use the `setSource` version that takes a field name and save a few characters. No big deal either way though.